cmd/exo/cmd/vm.go

```diff
@@ -37,6 +37,26 @@ func getSecurityGroup(vm *egoscale.VirtualMachine) []string {
 	return sgs
 }
 
+func getVolumeByVMID(vmID string) (*egoscale.Volume, error) {
+	volumes, err := cs.List(&egoscale.Volume{
```

Comment:
adding a `Get` on `Volume` would do pretty much the same a below wouldn't it?

Reply:
yes ! but `Get` want a `volumeID`. and when i asked to you you reply me the opposite

Reply:
If `Get` is dummy, make it smarter.

I probably lacked the proper context to answer you back then. I had an idea in the back of my brain to do `volumes, err := vm.Volumes()` and what thinking of that.

Sorry mate, to much on my plate.

Reply:
woww `volumes, err := vm.Volumes()`can be great ! So make `Get` smarter is better option !